internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/router/TestStreaming.java

```diff
@@ -384,7 +384,7 @@ private void verifyComputeResult(Map<String, GenericRecord> resultMap) {
     }
   }
 
-  @Test(timeOut = 5000)
+  @Test(timeOut = 30000)
```

Comment:
nit: maybe we can also make it a variable.

Reply:
I will close this since the timeout can be adjusted case by case.